strings/remove duplicates.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:test/test.dart';
+
+// function to remove duplicate in string
+String removeDups(String s) {
```

Comment:
you can change the function names to removeDuplicates that would be cleaner
and change the variable names like s and l to more meaningful variable names 
as it will make the code neater

Reply:
All in all, it looks good to me
just make the function naming and variable naming more better

Reply:
Is this already resolved?